test/specs/core.controller.tests.js

```diff
@@ -807,4 +807,107 @@ describe('Chart', function() {
 			]);
 		});
 	});
+
+	describe('controller.update', function() {
+		var chart;
+		var renderSpy;
+
+		beforeEach(function() {
+			chart = acquireChart({
+				type: 'line',
+				data: {
+					labels: ['A', 'B', 'C', 'D'],
+					datasets: [{
+						data: [10, 20, 30, 100]
+					}]
+				},
+				options: {
+					responsive: true
+				}
+			});
+
+			renderSpy = spyOn(chart, 'render');
+		});
+
+		describe('when _bufferedRender is true', function() {
+			beforeEach(function() {
+				chart._bufferedRender = true;
+			});
+
+			it('does not call render', function() {
+				chart.update({
+					duration: 800,
+					easing: 'easeOutBounce'
+				});
+
+				expect(renderSpy).not.toHaveBeenCalled();
+			});
+
+			it('builds _bufferedRequest with config properties', function() {
+				chart.update({
+					duration: 800,
+					easing: 'easeOutBounce'
+				});
+
+				expect(chart._bufferedRequest).toEqual({
+					duration: 800,
+					lazy: undefined,
+					easing: 'easeOutBounce'
+				});
+			});
+		});
+
+		describe('when _bufferedRender is false', function() {
+			beforeEach(function() {
+				chart._bufferedRender = false;
+			});
+
+			it('calls render with the config object', function() {
+				chart.update({
+					duration: 800,
+					easing: 'easeOutBounce'
+				});
+
+				expect(renderSpy).toHaveBeenCalledWith({
+					duration: 800,
+					easing: 'easeOutBounce'
+				});
+			});
+		});
+
+		describe('when using backwards compatibility', function() {
```

Comment:
Can we move that "compatibility" suite in [global.deprecations.tests.js](https://github.com/chartjs/Chart.js/blob/master/test/specs/global.deprecations.tests.js) as:

```javascript
describe('Deprecations', function() {
    describe('Version 2.7.0', function() {
        describe('Chart.Controller.update(duration, lazy)', function() {
            // ...
        });
        describe('Chart.Controller.render(duration, lazy)', function() {
            // ...
        });
    });

	describe('Version 2.6.0', function() {
    // ...
```